platform/admin/ui/src/main/webapp/js/views/installer/GuestClaims.view.js

```diff
@@ -374,7 +374,13 @@ define([
             this.submitData();
 
             //save the config
-            this.writeClaims(this.configObj.attributes.properties.attributes);
+            var claims = this.configObj.attributes.properties.attributes;
+            if (typeof claims.profile !== 'undefined' && claims.profile !== 'Default') {
+                view.navigationModel.set('modified', true);
+            } else {
+                view.navigationModel.set('modified', false);
+            }
+            this.writeClaims(claims);
```

Comment:
Here we handle the showing of the "restart required" screen after install, which occurs only if a non-default profile is selected. On the first submission, it is possible for `claims.profile` to be undefined, which means 'Default' was chosen.